explore/basic_threads.R

```diff
@@ -0,0 +1,15 @@
+library(Rcpp)
```

Comment:
drop file?

Reply:
`explore/` is in `.Rbuildignore`. I think it's useful to have some scratch work - type files that are also version controlled. Bit of a testing workbench for new features etc.